mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -909,6 +913,20 @@ The `scheduler` block supports:
   (Optional)
   The number of schedulers.
 
+The `triggerer` block supports:
+
+* `cpu` -
+  (Optional)
```

Comment:
These fields are marked Required in the schema, so they should be documented as such